testsuite/cluster-test/src/main.rs

```diff
@@ -512,7 +510,7 @@ impl ClusterTestRunner {
             // This assumes so far that event propagation time is << 1s, this need to be refined
             // in future to account for actual event propagation delay
             let events = self.logs.recv_all_until_deadline(deadline);
-            self.health_check_runner.run(&events);
+            self.health_check_runner.run(&events, &HashSet::new(), true);
```

Comment:
Here compiler gives a warning that we are not using `std::result::Result`. 

I guess we don't care about the returned Result here, we only want to use this to print healthcheck messages.

Reply:
Yeah, you can do something like `let _ignore = self. health_check_runner.run...`

Reply:
Updated